guides/source/testing.md

```diff
@@ -1405,7 +1405,7 @@ In order to test that your mailer is working as expected, you can use unit tests
 
 #### Revenge of the Fixtures
 
-For the purposes of unit testing a mailer, fixtures are used to provide an example of how the output _should_ look. Because these are example emails, and not Active Record data like the other fixtures, they are kept in their own subdirectory apart from the other fixtures. The name of the directory within `test/fixtures` directly corresponds to the name of the mailer. So, for a mailer named `UserMailer`, the fixtures should reside in `test/fixtures/user_mailer` directory.
+For the purposes of unit testing a mailer, fixtures are used to provide an example of how the output _should_ look. Because these are example emails, and not Active Record data like the other fixtures, they are kept in their own subdirectory apart from the other fixtures. The name of the directory within `test/fixtures` directly corresponds to the name of the mailer. So, for a mailer named `UserMailer`, the fixtures should reside in `test/fixtures/user_mailer` directory. By convention, each fixture has the name of an action, but you can add as many fixtures as you like.
```

Comment:
I assume the addition here is:
" By convention, each fixture has the name of an action, but you can add as many fixtures as you like. "

Can you let me know the motive behind adding it, not covered by other parts?